app/web/features/search/FilterDialog.stories.tsx

```diff
@@ -25,7 +24,7 @@ const Template: Story<any> = () => {
         setHostingStatusFilter={() => {}}
         completeProfileFilter={true}
         setCompleteProfileFilter={() => {}}
-        numberOfGuestFilter={undefined}
+        numberOfGuestFilter={""}
```

Comment:
why not undefined? I want to be able to have undefined, since it's the default value of the dropdown, plus it's what is sent to the server